{"name": "test_negative_delete_task_by_id", "status": "passed", "description": "Generate random task ID and try to delete it without permission, validate that received error message and code equal to expected", "steps": [{"name": "Delete task by ID: 859, expected status code: 401", "status": "passed", "steps": [{"name": "Create url with base url and endpoint: /task/859\nResult URL: https://api.clickup.com/api/v2/task/859", "status": "passed", "start": 1746890917880, "stop": 1746890917880}, {"name": "Send DELETE request to https://api.clickup.com/api/v2/task/859 with json=None and data=None, expected status code 401", "status": "passed", "start": 1746890917880, "stop": 1746890919475}], "start": 1746890917880, "stop": 1746890919475}], "start": 1746890917880, "stop": 1746890919475, "uuid": "69d83375-b971-4cdf-97cc-25f34fa3a017", "historyId": "111aa04c46da628e59bae88064c9bbd8", "testCaseId": "111aa04c46da628e59bae88064c9bbd8", "fullName": "tests.api.test_tasks.TestTasks#test_negative_delete_task_by_id", "labels": [{"name": "feature", "value": "Manage tasks"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "apachai"}, {"name": "thread", "value": "7984-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}